.github/workflows/tests_development.yml

```diff
@@ -30,20 +33,27 @@ jobs:
           --health-interval 10s
           --health-timeout 5s
           --health-retries 5
-
     steps:
     - uses: actions/checkout@v2
 
-    - name: Set up Python 3.6
-      uses: actions/setup-python@v2
+    - name: Set up Conda
```

Comment:
Since the conda install method is a temporary workaround I would prefer us to leave the test_development.yml workflow as it is (but using python 3.9 or the matrix above) and install and test using pip. 

But we definitely need to have tests covering the conda install method while it is in use so could you create a new job in this workflow `test_conda` that installs with conda. This way once the pip wheels for the necessary packages are updated to work with arm64 we can easily disable that workflow without needing to modify this one.